Sources/SnowManLogic/GameLogic/Guess.swift

```diff
@@ -0,0 +1,11 @@
+public class Guess {
+	public var currentGuess: String
+
+	public init(currentGuess: String) {
+		self.currentGuess = currentGuess
+	}
+
+	public func isFullWord() -> Bool {
+		return currentGuess.characters.count > 1
+	}
+}
```

Comment:
@zacharyjdavy please check. I separated the functionality between a guess and what it knows about itself, and manager managing multiple guesses